gradle/verification-metadata.xml

```diff
@@ -3,31 +3,27 @@
    <configuration>
       <verify-metadata>true</verify-metadata>
       <verify-signatures>false</verify-signatures>
-      <trusted-artifacts>
-         <trust file=".*-javadoc[.]jar" regex="true"/>
-         <trust file=".*-sources[.]jar" regex="true"/>
-      </trusted-artifacts>
```

Comment:
why removing those artifacts? they are needed for local project build

Reply:
fixed by https://github.com/pagopa/pagopa-payment-wallet-event-dispatcher-service/pull/26/commits/7599761a167d50c40ed27472c2b37661c58979ba